Userland/Libraries/LibWeb/XML/XMLDocumentBuilder.cpp

```diff
@@ -86,8 +86,15 @@ void XMLDocumentBuilder::element_start(const XML::Name& name, HashMap<XML::Name,
         MUST(m_current_node->append_child(node));
     }
 
-    for (auto const& attribute : attributes)
+    for (auto const& attribute : attributes) {
+        // https://www.w3.org/TR/xml-names/#ns-decl
```

Comment:
I couldn't find any better source for this... but this is what other browsers do apparently.

Reply:
This is fine, just link to the newer version ([Declaring Namespace](https://www.w3.org/TR/2006/REC-xml-names11-20060816/#ns-decl)) to avoid confusion, some behaviours _are_ gated behind xml 1.0 compat mode.